lib/staccato/adapter/net_http.rb

```diff
@@ -3,7 +3,11 @@
 module Staccato
   module Adapter
     module Net
-      class HTTP # The net/http Standard Library Adapter
+      # The net/http Standard Library Adapter
+      class HTTP
+
+        attr_reader :uri
```

Comment:
Is this reader used outside the adapter?

Reply:
I used it in the specs to verify collection vs validation URIs.  And I like to keep `attr_reader`s for this sort of thing for more "ergonomic" debug access.

Reply:
If it is, should the `Validate` adapter also have a uri reader?

Reply:
Nevermind, it delegates to the adapter.